go/chat/attachment.go

```diff
@@ -99,207 +192,78 @@ func DownloadAsset(ctx context.Context, log logger.Logger, params chat1.S3Params
 
 	// decrypt body
 	dec := NewSignDecrypter()
-	decBody := dec.Decrypt(tee, asset.Key, asset.VerifyKey)
-	if err != nil {
-		return err
+	var decBody io.Reader
+	if asset.Nonce != nil {
```

Comment:
Existing attachments.  I'm fine with not supporting them, although last time there was a GH issue from some non-keybase users who are using chat.  I'd rather not bump the version for adding a field at this point.